.circleci/config.yml

```diff
@@ -24,64 +23,64 @@ commands:
             fi
       - checkout
       - run:
-          name: Compress git objects 
+          name: Compress git objects
           command: git gc
       - save_cache:
           name: Git save cache
           key: source-v1-{{ .Branch }}-{{ .Revision }}
           paths:
             - ".git"
-  compress:
-    description: "Compress"
-    steps:
-      - run:
-          name: "Compress"
-          command: |
-            tar -cvf artifact.tar --exclude='README.md' --exclude='Dockerfile' --exclude='default.conf' * 
-      - run:
-          name: "Tag commit id as artifact identifer"
-          command: echo "${CIRCLE_SHA1}" > artifact-info.txt
-  upload_artifact:
-    description: "upload artifact to s3"
-    steps:
-      - s3/copy:    
-          from: artifact.tar
-          to: 's3://${CONTEXT_ARTIFACT_S3_BUCKET}/${CIRCLE_PROJECT_REPONAME}/'
-          aws-access-key-id: env_CONTEXT_ARTIFACT_S3_AWS_ACCESS_KEY_ID
-          aws-secret-access-key: env_CONTEXT_ARTIFACT_S3_AWS_SECRET_ACCESS_KEY
-          aws-region: env_CONTEXT_ARTIFACT_S3_AWS_REGION
-          arguments: '--metadata "{\"x-amz-artifact-id\": \"${CIRCLE_SHA1}\" }"'
-  upload_checksum:
-    description: "upload artifact checksum to s3"
-    steps:
-      - s3/copy:    
-          from: artifact-info.txt
-          to: 's3://${CONTEXT_ARTIFACT_S3_BUCKET}/${CIRCLE_PROJECT_REPONAME}/'
-          aws-access-key-id: env_CONTEXT_ARTIFACT_S3_AWS_ACCESS_KEY_ID
-          aws-secret-access-key: env_CONTEXT_ARTIFACT_S3_AWS_SECRET_ACCESS_KEY
-          aws-region: env_CONTEXT_ARTIFACT_S3_AWS_REGION
   docker_build:
     description: "Build Docker image"
     steps:
       - setup_remote_docker
-      - run: 
+      - run:
           name: Building docker image
           command: |
             docker build -t ${DOCKHUB_ORGANISATION}/binary-bitcoin-wallet:${CIRCLE_SHA1} -t ${DOCKHUB_ORGANISATION}/binary-bitcoin-wallet:latest .
   docker_push:
     description: "Push image to docker hub"
     steps:
-      - run: 
+      - run:
           name: Pushing Image to docker hub
           command: |
             echo $DOCKERHUB_PASSWORD | docker login -u $DOCKERHUB_USERNAME --password-stdin
             docker push ${DOCKHUB_ORGANISATION}/binary-bitcoin-wallet:${CIRCLE_SHA1}
             docker push ${DOCKHUB_ORGANISATION}/binary-bitcoin-wallet:latest
+  deploy_gh_pages:
+    description: "Deploy to static branches"
+    parameters:
+      target_branch:
+        type: string
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ./
+      - run:
+          name: Tag build
+          command: echo "<< parameters.target_branch >>-$(date -u +'%Y-%m-%dT%H:%M:%SZ')" > ./version
+      - run:
+          name: Install and configure dependencies
+          command: |
+            curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+            sudo apt install nodejs
```

Comment:
can we use circle node image option instead of installing nodejs from apt ? 
```
docker:
      - image: circleci/node
```

Reply:
It is not possible. We require Ruby image to build the project.